### ISSUE TITLE
Camera Calibration Data Inconsistency (for IR and Depth)

### ISSUE THREAD

**Issue by Spatiumensura** [code-iai/iai_kinect2#391]
If there is a set of individual camera calibration data (stored in the data directory), then this individual camera calibration data is loaded into the `cameraMatrixDepth` and `cameraDistortionDepth` only (see https://github.com/code-iai/iai_kinect2/blob/master/kinect2_bridge/src/kinect2_bridge.cpp#L640). The `cameraMatrixIr` and `cameraDistortionIr` matrices still hold the default calibration data from the device. Is this right and intended to do so?

**Comment by wiedemeyer** [comment]
Yes, it is intended like this, so that the SD color image fits the ir image, because the registration for the SD topics uses the sensor defaults.
